tests/conftest.py

```diff
@@ -15,3 +22,61 @@ def reset_torch_dynamo():
     # PyTorch caches models. Start a fresh compile for each parameter of the test case.
     torch._dynamo.reset()
     yield
+
+
+@pytest.fixture(autouse=True)
```

Comment:
does autouse apply it to lowering tests too?

Reply:
Checked on a call. Yes. 
We agreed it is ok for now. 
We agreed this will go away during the next step.